desktop-src/VSS/vshadow-tool-and-sample.md

```diff
@@ -74,7 +74,7 @@ This command creates a new shadow copy set.
 
 ## Importing a Shadow Copy Set
 
-**vshadow** \[OptionalFlags\] **-i=***File***.xml**
+**vshadow** \[OptionalFlags\] **-i=** *File* **.xml**
```

Comment:
Adding spaces within the filename might be confusing to readers. The mix of bold and italics is also hard to read. Would it work better if all of the examples were formatted as code? Something like this, using angle brackets to indicate parameters to be replaced:

```suggestion
`vshadow [OptionalFlags] -i=<File>.xml`
```

Reply:
@stardustman You're right. This is ugly, but my suggested change makes it look much better with code formatting: 

![image](https://github.com/user-attachments/assets/74265280-6203-4c97-bef0-06f045080bf5)

Reply:
Here's a larger sample of how the page could look with code formatting used instead of a mix of bold and italics.

![image](https://github.com/user-attachments/assets/8a755b78-a975-40ab-b8da-fb2ca365f747)

Reply:
I have a PR ready to re-format the page this way.

Reply:
ok. thanks.